src/pke/lib/scheme/bfvrns/bfvrns-leveledshe.cpp

```diff
@@ -180,10 +196,13 @@ Ciphertext<DCRTPoly> LeveledSHEBFVRNS::EvalMult(ConstCiphertext<DCRTPoly> cipher
     std::vector<DCRTPoly> cv1 = ciphertext1->GetElements();
     std::vector<DCRTPoly> cv2 = ciphertext2->GetElements();
 
-    size_t cv1Size    = cv1.size();
-    size_t cv2Size    = cv2.size();
-    size_t cvMultSize = cv1Size + cv2Size - 1;
-    size_t sizeQ      = cv1[0].GetNumOfElements();
+    size_t cv1Size           = cv1.size();
+    size_t cv2Size           = cv2.size();
+    size_t cvMultSize        = cv1Size + cv2Size - 1;
+    size_t sizeQ             = cv1[0].GetNumOfElements();
```

Comment:
Calling EvalAdd, EvalSub or EvalMult on plaintexts/ciphertexts of different levels does not always throw the exception of "called on NativeVectorT's with different parameters." This exceptions is thrown when the first ciphertext has all limbs, but the second one has fewer, but when the first ciphertext has fewer limbs and the second one has all limbs, it just outputs an incorrect result.

Reply:
I've added a separate issue for this: https://github.com/openfheorg/openfhe-development/issues/721 (to fix later).